src/components/page-configuration/change-admin-form.tsx

```diff
@@ -119,7 +146,7 @@ export class ChangeAdminForm extends Component<
         <Accordion.Item style={{ overflow: 'visible' }}>
           <Accordion.Header>
             <Accordion.HeaderTitle>
-              <Typography>Change administrators</Typography>
+              <Typography>Access control</Typography>
```

Comment:
Wrap both group selectors into the same form.
Only show one error message if save fails.

Something like this

```
      <Accordion className="accordion" chevronPosition="right">
        <Accordion.Item style={{ overflow: 'visible' }}>
          <Accordion.Header>
            <Accordion.HeaderTitle>
              <Typography>Access control</Typography>
            </Accordion.HeaderTitle>
          </Accordion.Header>
          <Accordion.Panel>
            <form
              className="grid grid--gap-medium"
              onSubmit={this.handleSubmit}
            >
              {this.props.modifyState === RequestState.FAILURE && (
                <div>
                  <Alert type="danger">
                    Failed to change access control. {this.props.modifyError}
                  </Alert>
                </div>
              )}
              <div className="grid grid--gap-medium">
                <AppConfigAdGroups
                  labeling={'Administrators'}
                  adGroups={
                    this.state.appRegistrationPatchRequest
                      .applicationRegistrationPatch.adGroups
                  }
                  handleAdGroupsChange={this.handleAdGroupsChange}
                />
                <AppConfigAdGroups
                  labeling={'Readers'}
                  adGroups={
                    this.state.appRegistrationPatchRequest
                      .applicationRegistrationPatch.readerAdGroups
                  }
                  handleAdGroupsChange={this.handleReaderAdGroupsChange}
                />
              </div>
              {this.props.modifyState === RequestState.IN_PROGRESS ? (
                <div>
                  <CircularProgress size={24} /> Updating…
                </div>
              ) : (
                <div>
                  <Button color="danger" type="submit">
                    Change access control
                  </Button>
                </div>
              )}
            </form>
          </Accordion.Panel>
        </Accordion.Item>
      </Accordion>
```